src/Modals/ModalSignUp/ModalSignUp.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz');
+
+.form_signUp{
+    z-index: 11;
+    position: absolute;
+    top: 35%;
+    left: 40%;
+    width: 300px;
+    height: 170px;
+    background: white;
+    border: 1px solid rgb(159, 247, 221);
+    border-radius: 5px;
+}
+
+.form_signUp_header{
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    margin-bottom: 15px;
+    background: rgb(159, 247, 221);
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+
+.signUp_username, .signUp_email{
+    width: 100%;
+    height: 40px;
+}
+
+.signUp_username{
+    margin-bottom: 10px;
+}
+.input_username, .input_email{
```

Comment:
you have a lot of rules that apply to both. Remember to use the trick where you can assign multiple class names to a single div. 

So a `signup_modal_label` class will have all these rules, then you can have more specific `signup_modal_label--username` for rules specific to username